doc/sphinx/source/quickstart/running.rst

```diff
@@ -4,138 +4,88 @@
 
 An
 `example recipe <https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/examples/recipe_python.yml>`_
 is available in the ESMValTool installation folder as
 ``examples/recipe_python.yml``.
 
-This recipe finds data from CanESM2 and MPI-ESM-LR for 2000 - 2002,
-extracts a single level (850 hPa), regrids it to a 1x1 degree mesh and runs
-a diagnostic script that creates some plots of Air temperature and
-precipitation flux. You can download the recipe from
-`github <https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/examples/recipe_python.yml>`_
-and save it in your project directory as (e.g.) ``recipe_python.yml``
-and then run ESMValTool with
+To run this recipe and automatically download the required climate data
+from ESGF to the local directory ``~/climate_data``, run
 
 .. code:: bash
 
-	esmvaltool run recipe_python.yml --synda-download
+	esmvaltool run examples/recipe_python.yml --offline=False
 
-The ``--synda-download`` option tells ESMValTool to use Synda to search for and download
+The ``--offline=False`` option tells ESMValTool to search for and download
 the necessary datasets.
-
-ESMValTool will also find recipes that are stored in its installation directory.
-A copy of the example recipe is shipped with ESMValTool as:
-``/path/to/installation/esmvaltool/recipes/examples/recipe_python.yml``.
-Thus, the following also works:
-
-.. code:: bash
-
-	esmvaltool run examples/recipe_python.yml
-
-Note that this command does not call Synda. The required data should thus be
-located in the directories specified in your user configuration file.
+If you have data available locally, you can run the tool without the
+``--offline=False`` argument (the default).
+Note that in that case the required data should be located in the directories
+specified in your user configuration file.
 Recall that the chapter :ref:`Configuring ESMValTool <config-user>`
 provides an explanation of how to create your own config-user.yml file.
 
-To get help on additional commands, please use
-
-.. code:: bash
-
-	esmvaltool --help
-
-It is also possible to get help on specific commands, e.g.
-
-.. code:: bash
-
-	esmvaltool run --help
-
-will display the help message with all options for the ``run`` command.
-
-There is a step-by-step description available in the
-`ESMValTool tutorial <https://esmvalgroup.github.io/ESMValTool_Tutorial/>`_
-on how to run your first recipe. It can be found
-`here <https://esmvalgroup.github.io/ESMValTool_Tutorial/04-recipe/index.html>`_.
+.. _recipes_command:
 
 Available diagnostics and metrics
 =================================
 
-See Section :doc:`Recipes <../recipes/index>` for a description of all
-available recipes.
-
-To see a list of installed recipes run
-
-.. code:: bash
+Although ESMValTool can be used just to simplify the management of data
+and the creation of your own analysis code, one of its main strengths is the
```

Comment:
this is a bit vague: I'd suggest using a phrasing a bit more explicit e.g. "can be used to download data and analyze it using ESMValCore's preprocessing modules". Also, "one of its main strengths" should probably be replaced with "its main purpose"

Reply:
Thanks, I updated the text